src/initialise.jl

```diff
@@ -570,12 +584,16 @@ where gamma_v (`gamma_profile`) is either specified in the vessel definition or
 assumed equal to `9` (plug-flow).
 """
 function computeViscousTerm(vessel_data :: Dict{Any,Any}, blood :: Blood)
-    if haskey(vessel_data, "gamma_profile")
+
+    if haskey(vessel_data,"Rc")                     # MODIFIED THIS LINE
+    return 2*blood.mu*blood.rho_inv*sqrt(30)
+    elseif haskey(vessel_data, "gamma_profile")
         gamma_profile = vessel_data["gamma_profile"]
     else
         gamma_profile = 9
+    return 2*(gamma_profile + 2)*blood.mu*blood.rho_inv
     end
-    return 2*(gamma_profile + 2)*pi*blood.mu*blood.rho_inv
+
 end
```

Comment:
@ibenemerito88  occhio che il return e' sempre dentro l'ultimo `else` (in teoria julia dovrebbe essere abbastanza intelligente da fare il return di `gamma_profile` anche per linea 591...ma non mi affiderei troppo a queste proprieta' esoteriche del linguaggio)


```julia
function computeViscousTerm(vessel_data :: Dict{Any,Any}, blood :: Blood)
    if haskey(vessel_data,"Rc")                     # MODIFIED THIS LINE
        return 2*blood.mu*blood.rho_inv*sqrt(30)
    elseif haskey(vessel_data, "gamma_profile")
        gamma_profile = vessel_data["gamma_profile"]
    else
        gamma_profile = 9
    end
    return 2*(gamma_profile + 2)*blood.mu*blood.rho_inv
end
```

Reply:
(sicuri che `pi` non ci vada? tutta la validazione della tesi e' stata fatta in quel modo)